data/import_method.py

```diff
@@ -1,16 +1,12 @@
 #!/usr/bin/env python3
-
 from bw2data.project import projects
 import bw2data
 import bw2io
+import sys
 
-# import functools
-
-PROJECT = "Food"
+PROJECT = sys.argv[1]
 # Agribalyse
-DATAPATH = "AGB3.1.1.20230306.CSV.zip"
-DBNAME = "Agribalyse 3.1.1"
-BIOSPHERE = DBNAME + " biosphere"
+BIOSPHERE = "Agribalyse 3.1.1 biosphere" if PROJECT == "Food" else "biosphere3"
```

Comment:
Should there be some kind of warning or help message if there's a typo instead of defaulting to textile?

Reply:
I'll remove that because I'm testing moving everything to biosphere3